src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private static final String BALANCE_MARKING = "b";
+    private static final String SUPPLY_MARKING = "s";
+    private static final String PURCHASE_MARKING = "p";
+    private static final String RETURN_MARKING = "r";
+    private Operation operation;
+    private Fruit fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public FruitTransaction.Operation getOperation(String letter) {
+        switch (letter) {
+            case BALANCE_MARKING :
+                return Operation.BALANCE;
+            case SUPPLY_MARKING :
+                return Operation.SUPPLY;
+            case RETURN_MARKING :
+                return Operation.RETURN;
+            default:
+                return Operation.PURCHASE;
+        }
+    }
+
```

Comment:
let's move this method inside the enum and let's not use switch case there 

look at the method fromString on screenshot
<img width="675" alt="Screenshot 2022-02-18 at 10 48 23" src="https://user-images.githubusercontent.com/33026471/154649301-d60410df-6344-4e08-8d14-207694aaafb0.png">

Reply:
Thanks, I couldn't  find a good way to realize this method )